scripts/scripts.js

```diff
@@ -142,16 +142,23 @@ export function decorateMain(main) {
   decorateVideoLinks(main);
 }
 
-function decorateTemplateAndTheme() {
-  aemDecorateTemplateAndTheme();
-  const { classList } = document.body;
+function decorateMode(element) {
+  const { classList } = element;
   if (classList.contains('light')) {
     classList.add('calcite-mode-light');
   } else if (classList.contains('dark')) {
     classList.add('calcite-mode-dark');
   }
 }
 
+export function decorateBlockMode(block) {
```

Comment:
I think the comment disappeared with the change, but why not export "decorateMode" directly? (as a block is also an element)

Reply:
replied above. We don't want it for every element. We stated that it should be page only, or block only. 

Not generalize it down to the level of element. Anyways, it is just semantics and consistency. 

(look reply above.)